test-suite/squidconf/time_units

```diff
@@ -15,17 +15,16 @@ dns_timeout 1 millisecond
 dns_timeout 0.001 second
 
 # maximum checks
```

Comment:
We should explain how these values are selected since they were selected poorly when introduced in #443 (necessitating this PR) and may have new selection problems during the first attempt in this PR.

```suggestion
# check support for the maximum value (expressed in each supported time unit)
# XXX: Use a 32-bit maximum to avoid breaking tests on 32-bit platforms.
# TODO: Generate these test lines based on the target build platform.
```